### PR TITLE
[Bug 1083914] Added Datepicker to Create announcement on l10n dashboard

### PR DESCRIPTION
**Pull request by anushbmx**
This will add date picker to the create announcement pop up.

### CODE REVIEW

**Comment by rehandalal** [comment]
test failure looks unrelated... restarting.

**Comment by anushbmx** [comment]
@rehandalal thanks for restarting I was wondering what made the test fail and it was unrelated. I couldn't test this on my local clone because I don know how to get that announcement button but did it a test on site with scratch pad after vesper gave me permission to do announcement in a locale.

**Comment by rehandalal** [comment]
there are a couple of issues with this.

the code should be in `wiki.dashboard.js` not `wiki.js`.

it should be wrapped with:
```js
if ($('body').is('.localization-dashboard')) {
...
}
```
also you will need to update line 5 in `dashboards/localization.html` to:
```
{% set scripts = ('wiki', 'jqueryui', 'wiki.dashboard') %}
```

**Comment by anushbmx** [comment]
Thanks @rehandalal I'll update it.

**Comment by anushbmx** [comment]
@rehandalal I have made the changes you mentioned, have a look.

**Comment by rehandalal** [comment]
you are still missing the change to `dashboards/localization.html` that i mentioned.

**Comment by anushbmx** [comment]
yiks let me get that too.

**Comment by anushbmx** [comment]
@rehandalal Done.

File: kitsune/dashboards/templates/dashboards/localization.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 {% from "dashboards/includes/macros.html" import print_readout, overview_section, product_choice_list, print_subscription_menu with context %}
 {% from "dashboards/includes/macros.html" import localization_sidebar_nav %}
 {% set title = _('[{locale}][{product}] Localization Dashboard')|f(locale=current_locale, product=_(product.title, 'DB: products.Product.title') if product else _('All Products')) %}
-{% set scripts = ('wiki', 'wiki.dashboard') %}
+{% set scripts = ('wiki', 'jqueryui', 'wiki.dashboard') %}
 {% set styles = ('kbdashboards', ) %}
```
Comment (rehandalal) [inline kitsune/dashboards/templates/dashboards/localization.html]: so the datepicker is working now but has no style. so you'll probably need to add 'jqueryui' here too.

**Comment by anushbmx** [comment]
@rehandalal I confirmed its working with styles.

**Comment by rehandalal** [comment]
looks good, thanks!

**Pull request merged**